_includes/docs/pe/user-guide/custom-menu.md

```diff
@@ -1,18 +1,141 @@
 {% assign feature = "Custom Menu" %}{% include templates/pe-feature-banner.md %}
 
-ThingsBoard Custom Menu feature allows you to extend ThingsBoard UI with custom menu items 
-and show/hide existing menu items.
-
-See video tutorial below for step-by-step instruction how to use this feature.
-
-<br/>
-<div id="video">  
-    <div id="video_wrapper">
-        <iframe src="https://www.youtube.com/embed/VSNZWl1NjWU" frameborder="0" allowfullscreen></iframe>
-    </div>
-</div> 
- 
-## Next steps
+* TOC
+{:toc}
+
+## Introduction
+
+The ThingsBoard Custom Menu feature allows you to extend ThingsBoard UI.
+You can add new and show/hide existing menu items.
+
+## Hide existing menu items
+
+To hide menu items in the ThingsBoard user interface, specify the menu items you want to hide in JSON data format in the "Custom Menu" window.
+
+Example of JSON data format to hide "Home" and "Alarms" menu items:
+
+```json
+{
+  "disabledMenuItems": [
+    "home",
+    "alarms"
+  ],
+  "menuItems": []
+}
+```
+{: .copy-code}
+
+Let's see how this works:
+
+{% assign addNewMenuItem = '
+    ===
+        image: /images/user-guide/white-labeling/custom-menu/custom-menu-add-new-menu-item-1.png,
+        title: Go to the "**White Labeling**" page -> "**Custom Menu**" tab. Menu items names that can be hidden are displayed in the empty "**Custom Menu**" window;
```

Comment:
Perhaps, we should list all the menu item backend names so the admin can view them in Documentation. 
The placeholder in the CM edit window disappears once the cursor is placed on the edit field. The user must pen down the required "names" elsewhere as they are intangible.